README.md

```diff
@@ -90,6 +90,37 @@ npm run mem_backend &
 npm run ft_test
 ```
 
+## Configuration
+
+If you want to specify an endpoint (other than localhost),
+you need to add it to your config.json:
+
+```shell
```

Comment:
Shouldn't it be `json`? (You'd need some editions to make your example valid JSON though).